pyro/distributions/ordered_logistic.py

```diff
@@ -0,0 +1,49 @@
+import torch
+from torch.distributions import constraints
+from pyro.distributions.torch import Categorical
```

Comment:
I would like to point out that for some reason when running tests, I could not get anything to work when using `from pyro.distributions import Categorical`; kept getting an error that the `Categorical` class didn't exist...

It seems that `pyro/distributions/__init__.py` is importing the `Categorical` class from `pyro/distributions/torch.py` though, so hopefully this doesn't cause any problems.